website/docs/r/app_service.html.markdown

```diff
@@ -149,7 +149,21 @@ Additional examples of how to run Containers via the `azurerm_app_service` resou
 
 * `remote_debugging_version` - (Optional) Which version of Visual Studio should the Remote Debugger be compatible with? Possible values are `VS2012`, `VS2013`, `VS2015` and `VS2017`.
 
-* `scm_type` - (Optional) The type of Source Control enabled for this App Service. Possible values include `None` and `LocalGit`. Defaults to `None`.
+* `scm_type` - (Optional) The type of Source Control enabled for this App Service. Defaults to `None`. Possible values are:
+  * `BitbucketGit`
+  * `BitbucketHg`
+  * `CodePlexGit`
+  * `CodePlexHg`
+  * `Dropbox`
+  * `ExternalGit`
+  * `ExternalHg`
+  * `GitHub`
+  * `LocalGit`
+  * `None`
+  * `OneDrive`
+  * `Tfs`
+  * `VSO`
+  * `VSTSRM`
```

Comment:
whilst this is a long list, we still tend to format this as:

```
`first`, `second`, `third` and `fourth`.
```

I'm going to push a commit to fix this (and also add it to the `app_service_slot` resource) - I hope you don't mind